benchmarks/benchmarks/bench_avx.py

```diff
@@ -106,8 +106,8 @@ def setup(self, bfuncname, stride, dtype):
         np.seterr(all='ignore')
         try:
             self.f = getattr(np, bfuncname)
-        except AttributeError:
-            raise NotImplementedError()
+        except AttributeError as e:
+            raise NotImplementedError() from e
```

Comment:
I think all of these should be
```suggestion
        except AttributeError:
            raise NotImplementedError(f"No ufunc {bfuncname} found") from None
```